src/screens/home/home.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { ScreenContainer } from '../../components/screen-container/screen-container';
+import { network } from '../../config/general';
+import { loadAssetsActions } from '../../store/assets/assets-actions';
+import { useAssetsSelector } from '../../store/assets/assets-selectors';
+import { useFirstAccountSelector } from '../../store/wallet/wallet-selectors';
+import { HomeStyles } from './home.styles';
+
+export const Home = () => {
+  const firstAccount = useFirstAccountSelector();
+  const assets = useAssetsSelector();
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
```

Comment:
function could be inlined

Reply:
Then it will be run after destroy

Reply:
`useEffect(() => void dispatch(....))` would do a trick

Reply:
Destructor calls only if you are returning a function